crates/pixi_manifest/src/task.rs

```diff
@@ -44,6 +56,55 @@ impl From<TaskName> for String {
     }
 }
 
+/// A task dependency with optional args
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize)]
+pub struct Dependency(pub TaskName, pub Option<Vec<String>>);
```

Comment:
```suggestion
pub struct Dependency {
pub task_name: TaskName
pub args: Option<Vec<String>>
}
```

I think this is more readable